Packs/Uptycs/Integrations/Uptycs/Uptycs.py

```diff
@@ -12,8 +12,8 @@
 from datetime import datetime, timedelta
 import requests
 from typing import List
-from signal import signal, SIGPIPE, SIG_DFL
-signal(SIGPIPE, SIG_DFL)
+from signal import signal, SIGPIPE, SIG_DFL  # type: ignore[no-redef]
+signal(SIGPIPE, SIG_DFL)  # type: ignore[operator]
```

Comment:
Tal added an import to signal to CommonServerPython yesterday.
option1:
move the CommonServerPython/demistomock imports after the integration imports.

or option2: (riding on the import from there)
```suggestion
signal.signal(signal.SIGPIPE, signal.SIG_DFL)  # type: ignore[operator]
```